### PR TITLE
Add plusMillis function and sample test cases

### PR DESCRIPTION
**Pull request by wiiitek**
I have prepared some unit tests computing new time based on start time and time passed.

We need to provide implementation for [`plusMillis(long millis)` function](https://github.com/The-Coobaz/crazyclock/pull/44/files#diff-1edf79482afffe24201dbbf9d7e8b9ef92f80f6ca00523ba3866a1c3fcf1446fR26).

### CODE REVIEW

File: src/FakeTime/FakeTimeTest.ino [hunk 1]
```diff
@@ -53,6 +53,58 @@ test(should_handle_seconds_errors_correctly) {
...
+
+  // when
+  tested.plusMillis(12000);
+  tested.formatTime(formattedTimeBuffer);
+
+  // then
+  assertEqual(formattedTimeBuffer, "17:00:12");
+}
+
+test(should_correctly_add_five_minutes_and_33_seconds) {
```
Comment (Qbunjo) [inline src/FakeTime/FakeTimeTest.ino:82]: Nice. How do you count that?

**Review by Qbunjo — APPROVED** [review]

File: src/FakeTime/FakeTimeTest.ino (hunk 1 above)
Comment (wiiitek) [reply src/FakeTime/FakeTimeTest.ino:82]: To calculate seconds:

```c
long seconds = millis / 1000;
```

then from the number of seconds we need to calculate *minutes* part and *seconds* part:

```c
long seconds = (millis / 1000) % 60;
long minutes = seconds / 60;
```

and we need to somehow extend this approach for hours...

Comment (Qbunjo) [reply src/FakeTime/FakeTimeTest.ino:82]: maybe
`long hoursToAdd = minutesToAdd / 60;`

File: src/FakeTime/FakeTime.h [hunk 2]
```diff
@@ -16,6 +16,12 @@ class FakeTime {
   FakeTime(int h, int m, int s);
 
   int formatTime(char *targetBuffer);
+
+  /**
+   * Updates the Fake time so that it show the hour, minute and second
+   * in future (after given milliseconds value).
+   */
+  void plusMillis(long millis);
```
Comment (Qbunjo) [inline src/FakeTime/FakeTime.h:24]: What should this function do?
If its time deviation from rtc, shouldn't it be like void FakeTime(int h, int m, int s, int deviation,bool changeFromRtc)? As every time we run FakeTime we should know:
1. If the time was deviated from RTC,
2. If yes, what is the deviation?

**Review by Qbunjo — APPROVED** [review]

**Pull request closed without merging**